lego/apps/events/tests/test_payments.py

```diff
@@ -20,8 +20,8 @@ class StripePaymentTestCase(BaseAPITestCase):
     fixtures = [
         "test_abakus_groups.yaml",
         "test_companies.yaml",
-        "test_events.yaml",
         "test_users.yaml",
+        "test_events.yaml",
```

Comment:
What's up here? Was the previous ordering causing issues when loading fixtures for the tests?

Reply:
Yup. `2017/11/13 21:52:34` was the last time we executed the stripe tests :tm:

Reply:
It worked on release, so why would we ever test it again??? 🙃

Reply:
Hahaha. Stripe has a quite stable API, so it shouldn't be a huge problem :laughing: